docs/frontend/introduction-to-less.md

```diff
@@ -0,0 +1,112 @@
+# Indtroduction to LESS
+This document describes behavior of CSS pre-processor [LESS](http://lesscss.org/).
+
+## Import files
+In Shopsys Framework we implement `LESS` by dividing styles into many `LESS components`. Each component has its own file. Filename and component name are the same. We use `@import` command for joining all components contents into one compiled `CSS` file.
+
+Here we can see syntax of `@import` command.
+```less
+@import 'path/to/directory/component-filename.less';
+```
+
+### Usage of @import command
+Best way how to import all related files is to create one file, for example named as `main.less`. This file will contain only `@import` commands. Keep in mind where you place this file. Imported path depends on where this file is placed.
+
+#### Unexpected behavior
+There is one thing which you should keep in mind. When you try to import file which does not exist in given file path, compiler will try to find missing file in root directories of files, where is used `@import`.
```

Comment:
```suggestion
There is one thing which you should keep in mind. When you try to import a file which does not exist in the given filepath, the compiler will try to find missing file in root directories of files, where is used `@import`.
```

Reply:
What do you want to say by "root directories of files, where is used `@import`"? There are more of them? Or only one root directory?

Reply:
This rule is applied for every file where is used `@import`.